code/game/objects/items.dm

```diff
@@ -196,6 +196,8 @@
 		if(src == user.get_inactive_hand())
```

Comment:
Also, I just realized that I didn't write this. It was in here already for handling wielding.
How does wielding work with multiple hands?